regulations/static/regulations/js/source/views/main/preamble-view.js

```diff
@@ -46,6 +47,21 @@ var PreambleView = ChildView.extend({
     );
   },
 
+  openCitation: function(e) {
+    var $target = $(e.currentTarget);
+    var hash = $target.attr('href');
+    var id = $target.attr('data-section-id');
+    var options = {};
+    var type = this.options.type;
+    var section = helpers.parsePreambleCitationId(hash, type);
+
+    if (id) {
+      e.preventDefault();
+    }
+
+    MainEvents.trigger('section:open', section, options, type);
```

Comment:
Curious, should this trigger go inside that `if`? Otherwise the browser will start an AJAX call (possibly failing) and switch pages before it completes?